internal/action/sync.go

```diff
@@ -128,6 +136,12 @@ func (s *Action) sync(ctx context.Context, store string) error {
 
 // syncMount syncs a single mount.
 func (s *Action) syncMount(ctx context.Context, mp string) error {
+	if as := config.FromContext(ctx).GetM(mp, "core.autosync"); as == "false" {
```

Comment:
Why not a `config.Bool(ctx, "core.autosync")` here?

Reply:
Added a comment.